src/test/ui/suggestions/enum-method-probe.fixed

```diff
@@ -0,0 +1,42 @@
+// run-rustfix
+#![allow(unused)]
+
+struct Foo;
+
+impl Foo {
+    fn get(&self) -> u8 {
+        42
+    }
+}
+
+fn test_result_in_result() -> Result<(), ()> {	
+    let res: Result<_, ()> = Ok(Foo);
+    res?.get();
+    //~^ ERROR no method named `get` found for enum `Result` in the current scope
+    //~| HELP use the `?` operator
+    Ok(())
+}
+
+fn test_result_in_plain() {	
+    let res: Result<_, ()> = Ok(Foo);
+    res.expect("REASON").get();
```

Comment:
should this just fixup to `.unwrap`? I kinda prefer `.expect` just because it feels like better practice out of the two.

Reply:
I know what you mean, expect is better style, but unwrap ends up looking cleaner... I can be convinced either way.